Android/app/src/main/java/objects/Contributor.java

```diff
@@ -0,0 +1,57 @@
+package objects;
+
+import com.google.gson.annotations.SerializedName;
+
+public class Contributor {
```

Comment:
we can benefit from the kotlin data class beauty, and create a new data class in the ApiResources.kt file like:
data class Contributor(@SerializedName("avatar_url") var avatarUrl: String,
                             @SerializedName("contributions") var contributions: String, ......... ) 
A data class in Kotlin automatically creates getters and setters, and more. You can check https://kotlinlang.org/docs/reference/data-classes.html

Reply:
Well, generally, i dont have experience with Kotlin. But, I will figure it out.
I will add this in the next commit

Reply:
Even I don't have experience with Kotlin, so can't say

Reply:
So, what if we added it as a new issue?